build.sbt

```diff
@@ -39,8 +37,7 @@ lazy val buildSettings = Seq(
         <email>dev-role@allenai.org</email>
       </developer>
     </developers>,
```

Comment:
I think `common` only has this because it used to be published to Maven, and Maven requires it. I don't think we should have this. It adds nothing but noise.